frontend/src/components/general/form/labeled_toggle/labeled_toggle.svelte

```diff
@@ -0,0 +1,11 @@
+<script>
```

Comment:
do you think this is the best way to split it up? personally i think its wayyy to split up and the logic can and should be reused in the parent component. then to have a single one just pass 1 value in the array

Reply:
yeah I was considering that too, I did it this way so that you could control the padding of the entire list seperately from the padding of each element. I dont really care either way, I dont really see an issue with splitting stuff up. I'll combine these into one for you though if you want